clang/lib/Gold/GoldElaborator.cpp

```diff
@@ -4533,7 +4710,7 @@ void Elaborator::elaborateExternAttr(Declaration *D, const Syntax *S,
   }
   // Checking a few things.
   if (isa<CallSyntax>(S)) {
-    llvm_unreachable("extern(\"C\") not implemented yet");
+    llvm_unreachable("This should never occur.");
```

Comment:
generally speaking, developer errors (that only occur on our part) should be assertions rather than unreachables, since they get compiled out. unreachables are more for users who intentionally try to put the compiler into weird states.